### PR TITLE
Convert naming state stores / internal topics, and multiple event types tutorials

### PR DESCRIPTION
**Pull request by davetroiano**

### CODE REVIEW

File: naming-changelog-repartition-topics/kstreams/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+Since the filter comes before the `count()` aggregation, the number in the state store and changelog / repartition topics will go from `0000000002` to `0000000003`.
+
+The upshot of this is that, when we run the application with the updated topology, the counts will start over from zero rather than start from where they left off.
+
+## Naming state stores and internal topics
+
+How can we reuse state stores and interrnal topics in the face of topology changes? There are a few places in the Kafka Streams DSL where we can control the names of the internal topics and state stores:
+
```
Comment (bbejeck) [inline naming-changelog-repartition-topics/kstreams/README.md:49]: Maybe work in the `Named` object in here somewhere that's what you'll use most of the time to name something

File: multiple-event-types-protobuf/kafka/src/main/proto/customer-event.proto [hunk 2]
```diff
@@ -3,15 +3,15 @@ syntax = "proto3";
 package io.confluent.developer.proto;
 
 import "purchase.proto";
-import "page-view.proto";
+import "pageview.proto";
 
 option java_outer_classname = "CustomerEventProto";
```
Comment (bbejeck) [inline multiple-event-types-protobuf/kafka/src/main/proto/customer-event.proto:8]: For all the protobuf files I'd add `option java_multiple_files = true` that way you can reference them in the Java code just by the class name and not the awkward Outerclass.ClassName.  

I wasn't aware of that when I did this originally

**Review by bbejeck — COMMENTED** [review]
LGTM I just have 2 comments

File: multiple-event-types-protobuf/kafka/src/main/proto/customer-event.proto (hunk 2 above)
Comment (davetroiano) [reply multiple-event-types-protobuf/kafka/src/main/proto/customer-event.proto:9]: done!

File: naming-changelog-repartition-topics/kstreams/README.md (hunk 1 above)
Comment (davetroiano) [reply naming-changelog-repartition-topics/kstreams/README.md:49]: added a note about this

**Review by bbejeck — APPROVED** [review]
LGTM!

**Pull request merged**